### ISSUE TITLE
Ice Dragon Breath:  More Features pls add really cool

### ISSUE THREAD

**Issue by Frosty8-0** [AlexModGuy/Ice-and-Fire-Suggestions#167]
Ice Dragon breath should be able to:
-  Freeze water
-  Freeze mobs.  This could be done by giving the mob effected extreme slowness, or completely freezing them in ice.  (freezing them in ice would be cooler, not only literally but because they could suffocate).
-  It would also be cool if you could progressively freeze larger mobs, like clipping a fire dragons tail with ice so that it flies slower, and being able to keep shooting it to continue freezing them until they are entirely frozen
-  Put out dragon fire and normal fire
-  Suffocate mobs frozen in ice or covered in snow
*More may come

**Comment by ZombieEnderman5** [comment]
Not bad! (Discord IGN is DeadlyBlueApples, you might remember me)

**Comment by Frosty8-0** [comment]
@Alex-the-666 Please look at my list and consider adding them pls pls pls

**Comment by TacoLatte** [comment]
Good idea... The ice dragons are being slightly overpowered by the fire dragons right now to be honest... Also am I the only one that feels like the title is a pun? No just me? Ok then...
